src/VisionModel/vgg.jl

```diff
@@ -0,0 +1,92 @@
+include("convnb.jl")
+
+export vgg11, vgg11bn, vgg13, vgg13bn, vgg16, vgg16bn, vgg19, vgg19bn
+
+const configs = Dict(:A => [(64,1) (128,1) (256,2) (512,2) (512,2)],
+                     :B => [(64,2) (128,2) (256,2) (512,2) (512,2)],
+                     :D => [(64,2) (128,2) (256,3) (512,3) (512,3)],
+                     :E => [(64,2) (128,2) (256,4) (512,4) (512,4)])
+
+# Build a VGG block
+#  ifilters: number of input filters
+#  ofilters: number of output filters
+#  batchnorm: add batchnorm (see below for the problem of biases in Conv)
+function vgg_block(ifilters, ofilters, depth, batchnorm)
+  k = (3,3)
+  p = (1,1)
+  i = Flux.glorot_uniform
+  layers = []
+  for l in 1:depth
+    if batchnorm
+      # Conv with BatchNorm must have no biases, not possible with Flux v0.10.4, however it seems
+      # available in Flux#master
+      push!(layers, ConvNB(k, ifilters=>ofilters, pad=p, init=i))
```

Comment:
Can you swap `ConvNB` out for `Conv` in Flux#master?

Reply:
As soon as I have a clean Flux#master on my system I remove my workaround, or if you prefer I can remove ConvNB and replace it with Conv + biases